rubygem-foreman_chef/rubygem-foreman_chef.spec

```diff
@@ -96,6 +94,11 @@ exit 0
 %doc %{gem_instdir}/README.md
 %{gem_instdir}/Rakefile
 
+%posttrans
+%foreman_db_migrate
```

Comment:
missing db:seed here (removed from line 79), a %foreman_db_seed macro should be available